framework/src/physics/PhysicsBase.C

```diff
@@ -29,17 +29,17 @@ PhysicsBase::validParams()
   params.addParam<MooseEnum>(
       "transient", transient_options, "Whether the physics is to be solved as a transient");
 
+  params.addParam<bool>("verbose", false, "Flag to facilitate debugging a Physics");
+
+  // Numerical solve parameters
   params.addParam<std::vector<SolverSystemName>>(
       "system_names",
       {"nl0"},
       "Name of the solver system(s) for the variables. If a single name is specified, "
       "that system is used for all solver variables.");
-
-  MooseEnum pc_options("default none", "none");
+  MooseEnum pc_options("default defer_to_other_physics", "defer_to_other_physics");
```

Comment:
can try to deprecate none already actually since my TMAP pr is not merged

Reply:
I don't think `none` is so bad. It describes a lot of the current use cases of physics right? There's not a single input with multiple physics merged into the MOOSE herd is there?

Reply:
If I'm right about the last question, then right now having the default enum be `defer_to_other_physics` doesn't seem like the best choice since it doesn't conceptually describe any of our inputs

Reply:
I can re-instate none later then haha. We can have as many options as we want to not add the preconditioning. 

>  There's not a single input with multiple physics merged into the MOOSE herd is there?

nope that s what the other PR enables though

Reply:
> it doesn't conceptually describe any of our inputs

it's transparent though. "defer_to_other..." will always mean "this object is not taking care of preconditioning"

Reply:
well now I'm thinking just `defer` is the best name. Because I assume in a lot of physics inputs we are deferring to the `Preconditioning` block

Reply:
If you go with plain `defer`, then I think dropping `none` makes total sense

Reply:
I ll do this now